logstash-core/spec/logstash/plugin_spec.rb

```diff
@@ -120,4 +120,33 @@ class LogStash::Filters::Stromae < LogStash::Filters::Base
       end
     end
   end
+
+  describe "subclass initialize" do
```

Comment:
Thinking about it more, we need a shared example similar to the shuts-down one that all plugins run because we can't otherwise guarantee that an author does not modify the params if they override `initialize`.